Xamarin.Essentials/Types/Location.shared.cs

```diff
@@ -39,6 +39,14 @@ public Location(double latitude, double longitude, DateTimeOffset timestamp)
             Timestamp = timestamp;
         }
 
+        public Location(double latitude, double longitude, double altitude)
```

Comment:
Is there a reason for this new constructor?

Reply:
Well, IMHO there is as much a reason for this as for those with (latitude, longitude) or (latitude, longitude, timestamp) parameters. None of them is strictly necessary, since the same can be accomplished also via an object initializer. But all of them can be handy, depending on the specific application.

It seems that the current state of Xamerin.Essentials.Location is mostly tailored towards 2D navigation-type applications, but there certainly are many other applications for the Location class, in particular in 3D, where a location is not complete without the altitude information. That is the specific use case of the new constructor.

Btw, I just noticed that I forgot to include documentation for the new constructor. I will do that, if we can agree that it is a useful addition.

Reply:
I think a simpler constructor is not bad if this type is to be used anywhere outside of the library. And, I am sure it is, so adding it is a good thing.